pkg/sql/sqlbase/structured.go

```diff
@@ -1424,9 +1422,11 @@ func (desc *MutableTableDescriptor) AllocateIDs() error {
 	if desc.ID == 0 {
 		desc.ID = keys.MinUserDescID
 	}
-	err := desc.ValidateTable()
+	// TODO(mgartner): How can I validate this without importing validator?
+	// err := validator.ValidateTable(desc.TableDesc())
+	// return err
```

Comment:
Looks like there is one place within `sqlbase` where `ValidateTable` is called... Any ideas on how to get over this last hurdle?